airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -50,3 +50,8 @@
   dockerRepository: airbyte/destination-mssql
   dockerImageTag: 0.1.3
   documentationUrl: https://docs.airbyte.io/integrations/destinations/mssql
+- destinationDefinitionId: 3986776d-2319-4de9-8af8-db14c0996e72
+  name: Oracle
```

Comment:
I think we should call it `Oracle (Alpha)` because it doesn't support normalization yet

Reply:
Done.